### ISSUE TITLE
kopy.io needs a DELETE ability

### ISSUE THREAD

**Issue by dch** [Komodo/KomodoEdit#1030]
This doesn't have to be from komodo itself, even a `curl ...` thing would be fine. I've fat-fingered several times stuff which I'd prefer not to have to wait for it to expire.

**Comment by Naatan** [comment]
Agreed. Until it does I suggest you use client side encryption if you tend to share sensitive information.

**Comment by dch** [comment]
It's the accidental cmd-K that gets me in a sensitive file (I am prone to RSI so not a great typist), not a pre-planned "oops is that in there". Client side encryption is a helpful default though.

**Comment by dch** [comment]
I just got hit by this again. Please please either provide a DELETE capability or encryption, or a prompt or something. I disabled the key shortcut completely.

**Comment by Defman21** [comment]
Kopy.io is based on Haste (as About window says). I've implemented this for Haste and once it will be merged to it, I think Nathan will merge this change as well to Kopy.

https://github.com/seejohnrun/haste-server/pull/125
